client/src-worker/turbo-resolver/resolver.js

```diff
@@ -99,7 +99,7 @@ export class Resolver {
     }
 
     if(!version){
-      throw new Error(`jsdelivr has not synced "${registryPackage.name}": "${requestedVersion}".`);
+      throw new Error(`npm package "${registryPackage.name}" was not found with requested version: "${requestedVersion}".`);
```

Comment:
You are correct, here got nothing to do with jsdelivr.

It should be `registry.npmjs.cf` out of sync, not jsdelivr (which I guessed and mistakenly put the error message).

The scenario is for aurelia 2 alpha release, we have a new dev release for every PR merge.
The user app starts resolving `"aurelia": "dev"`, which resolves to `"aurelia": "0.7.0.sometimestamp"`, it then recursively asks resolving deps like `"@aurelia/kernel": "0.7.0.same-stamp"`, and registry.npmjs.cf could not find `"0.7.0.same-stamp"` in `"@aurelia/kernel"`, likely because that it synced `"aurelia"` registry, but not `"@aurelia/kernel"` yet.

I will update browser-frame error according, or find a way to mitigate this issue.

On the other hand, it seems jsdelivr is very good at syncing with npm.

Reply:
I think I will migrate the turbo-resolver to use jsdelivr, so no more out of sync issue. Probably faster too.

https://github.com/jsdelivr/data.jsdelivr.com#list-package-versions

Reply:
Seems very easy based on the data shape. I will make the change after merge.

Reply:
The good news is you need one less private server, but you need to clone jsdelivr data APIs too.

Reply:
My long term goal is to replace this copied turbo-resolver from StackBlitz, because it doesn't actually replicate npm resolving result in complex situation (well, complex is the norm in any real app deps).

Only when I got enough time to study this topic...